src/blockchain_db/lmdb/db_lmdb.cpp

```diff
@@ -1367,6 +1389,10 @@ void BlockchainLMDB::open(const std::string& filename, const int db_flags)
   mdb_set_dupsort(txn, m_output_amounts, compare_uint64);
   mdb_set_dupsort(txn, m_output_txs, compare_uint64);
   mdb_set_dupsort(txn, m_block_info, compare_uint64);
+  if (!(mdb_flags & MDB_RDONLY))
+    mdb_set_dupsort(txn, m_txs_prunable_tip, compare_uint64);
+  mdb_set_dupsort(txn, m_txs_prunable, compare_uint64);
+  mdb_set_dupsort(txn, m_txs_prunable_hash, compare_uint64);
```

Comment:
It should be mdb_set_compare, but it looks like those tables could be made dupsort, yes, as well as the existing one, it might save some more space. I'll change to _compare for now and think about it afterwards.